Makefile

```diff
@@ -35,3 +55,21 @@ unit-test:
 
 build: $(SOURCES)
 	go build $(GOBUILDFLAGS) -ldflags="$(GOLDFLAGS)" -o ./update-service-operator ./
+
+# Generate manifests e.g. CRD, RBAC etc.
+manifests:
+	controller-gen $(CRD_OPTIONS) rbac:roleName=updateservice-operator webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+
+# Generate bundle manifests
+gen-for-bundle:
+	operator-sdk generate kustomize manifests -q
+
+# Generate bundle and metadata, then validate generated files.
+bundle:
```

Comment:
SDK's scaffolded `bundle` [depends on `manifests`][1].  With your current split, I'd expect `bundle` to depend on `gen-for-bundle`, and then either `bundle` or `gen-for-bundle` (I'm not sure which) to depend on `manifests`.

[1]: https://github.com/openshift/cincinnati-operator/blob/0731f08b2fbd3989b4642ff55c6a216e97d4c990/Makefile#L111

Reply:
Again, I left some of that out until we know what we the developers run and provide through git to the build tools. The Makefile as generated seems to be very much oriented towards a developer running everything with no consideration or expectation of an official build. 

In general though, I preferred not having the dependency to kick off manifests and running it myself explicitly.